serialisation/src/main/java/org/ehrbase/openehr/sdk/serialisation/flatencoding/std/marshal/StdFromCompositionWalker.java

```diff
@@ -75,13 +78,24 @@ protected void postHandle(Context<Map<String, Object>> context) {
         Class<? extends RMObject> aClass = context.getRmObjectDeque().peek().getClass();
 
         List<? extends MarshalPostprocessor<? extends RMObject>> postprocessor = findPostprocessors(aClass);
+        WebTemplateNode currentNode = context.getNodeDeque().poll();
+        WebTemplateNode parentNode = context.getNodeDeque().peek();
+        context.getNodeDeque().push(currentNode);
+
+        if ((parentNode == null)
+                || (!(currentNode != null && currentNode.getId().equals("name")) && isLocatable(parentNode))) {
```

Comment:
currentNode.getId().equals("name")) && isLocatable(parentNode)

pull that in a method isNameAttribute